components/ws-manager-bridge/src/bridge.ts

```diff
@@ -601,11 +602,17 @@ export class WorkspaceManagerBridge implements Disposable {
 
     protected async markWorkspaceInstanceAsStopped(ctx: TraceContext, info: RunningWorkspaceInfo, now: Date) {
         const nowISO = now.toISOString();
-        info.latestInstance.stoppingTime = nowISO;
+        if (!info.latestInstance.stoppingTime) {
+            info.latestInstance.stoppingTime = nowISO;
+        }
         info.latestInstance.stoppedTime = nowISO;
         info.latestInstance.status.phase = "stopped";
```

Comment:
Maybe `markWorkspaceInstanceAsStopped` should also set a `failed` condition, or something else that signals that there might not be a backup?

E.g. when I test the current code, any running instance is successfully marked as stopped. However, it's impossible to restart the instance:

<img width="600" alt="Screenshot 2022-09-15 at 14 07 14" src="https://user-images.githubusercontent.com/599268/190399815-d82bca5d-2989-4991-ab1a-c44dbc0ccdc0.png">

Reply:
@jankeromnes There is a condition that signals that there is a successful backup. But it only (re-)stabilized recently, and we do not use it, yet: https://github.com/gitpod-io/gitpod/issues/12387#issuecomment-1243071849